docs/content/features/exploiters/log4shell.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Log4Shell Exploiter"
+draft: false
+description: "Exploits the Log4Shell vulnerability"
+tags: ["exploiter", "log4shell", "log4j", "cve-2021-44228"]
+pre: "<i class='fa fa-mug-hot'></i> "
+---
+
+{{% notice info %}}
+The Log4Shell exploiter exploits [CVE-2021-44228](
+https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-44228).
+{{% /notice %}}
+
+## Apache Log4j
+
+[Apache Log4j](https://logging.apache.org/log4j/) is an open-source Java
+logging framework that is used to log messages within software, including
+information like error messages and user inputs in a program.
```

Comment:
Maybe some information about how many software packages use it, making them vulnerable.